lib/src/react_client/dart2_interop_workaround_bindings.dart

```diff
@@ -0,0 +1,14 @@
+@JS()
+library react_client.src.dart2_interop_workaround_bindings;
```

Comment:
we should probably add a comment explaining why this is needed, preferably with a link to the package:js or SDK issue if we create one